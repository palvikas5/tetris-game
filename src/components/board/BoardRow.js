import React from "react";
import { BOARD_SIZE } from "../../utils/constants";

const BoardRow = props => {
  return (
    <React.Fragment>
      {[...Array(BOARD_SIZE.col)].map((e, colIndex) => {
        const key = `${props.row}-${colIndex}`;
        return <div key={key} className="board-cell" />;
      })}
    </React.Fragment>
  );
};

export default BoardRow;
