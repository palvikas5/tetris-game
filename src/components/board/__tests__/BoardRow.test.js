import React from "react";
import { render } from "@testing-library/react";
import BoardRow from "../BoardRow";

describe("<BoardRow />", () => {
  it("should render successfully", () => {
    const { container } = render(<BoardRow />);
    expect(container).toMatchSnapshot();
  });
});
