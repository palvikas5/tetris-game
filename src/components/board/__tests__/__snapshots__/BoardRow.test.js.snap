// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BoardRow /> should render successfully 1`] = `
<div>
  <div
    class="board-cell"
  />
  <div
    class="board-cell"
  />
  <div
    class="board-cell"
  />
  <div
    class="board-cell"
  />
  <div
    class="board-cell"
  />
  <div
    class="board-cell"
  />
  <div
    class="board-cell"
  />
  <div
    class="board-cell"
  />
</div>
`;
